src/app/cli/main.cpp

```diff
@@ -89,18 +89,25 @@ static void HandleSignalInterrupt()
 
 int main(int argc, const char *argv[])
 {
-    Error  error;
-    Config config;
+    Error       error;
```

Comment:
may enhance to use getopt in future PRs.

Reply:
I see DSR's PR https://github.com/openthread/ot-commissioner/pull/202 already changed to use `getopt` so let that PR do the change.